code/native/mpris.py

```diff
@@ -178,6 +178,16 @@ def LoopStatus(self, value):  # noqa: N802
         if not self.CanControl:
             raise ValueError
 
+    @property
+    def Volume(self):
+        return self._volume
+
+    @Volume.setter
+    def Volume(self, value):
```

Comment:
Let's also add
```
if not self.CanControl:
    raise ValueError
```
here, before setting self._volume. This is according to the MPRIS specification.